server/routes/tasks.routes.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const controller = require("../controllers/tasks.controller");
+const validators = require("../validators/tasks.validator");
+
+const router = express.Router();
+
+router.get("/", controller.readTasks);
+
+router.post("/", controller.createTask);
```

Comment:
Why you don't use validation here?

Reply:
Because those paths do not need Joi validation. Updating a task only need Joi.

Reply:
No, it is not true. For creating task you also should validate body